tests/fixtures/tests.py

```diff
@@ -520,6 +521,14 @@ def test_loading_and_dumping(self):
             natural_foreign_keys=True,
         )
 
+    def test_deeply_nested_elements(self):
+        """Text inside deeply-nested tags is skipped."""
+        management.call_command(
+            "loaddata", "invalid_deeply_nested_elements.xml", verbosity=0
+        )
+        person = Person.objects.get(pk=1)
+        self.assertEqual(person.name, "Django")  # not "Django pony"
```

Comment:
I find that surprising. I'd expect `Django <em>pony</em>` or a `SuspiciousOperation`.

Reply:
Sure thing -- no rush!